scripts/version.sh

```diff
@@ -37,7 +37,7 @@ if [ -z "$VERSION_K8S" ]; then
     VERSION_K8S="v0.0.0"
 fi
 
-VERSION_CNIPLUGINS="v0.8.6-k3s1"
+VERSION_CNIPLUGINS="v0.9.1-k3s1"
```

Comment:
If this hasn't been called out already we should make an effort to avoid including this a patch release so as to avoid bumping the CNI plugins to a new minor version for the same minor version of k3s.

Reply:
... and of course I see that k3s-io/k3s#1405 is slated for 1.22.2. Welp, the earlier in the release series the better I guess. Should we consider bumping the CNI plugins to 0.9.x for k3s 1.22.1 to set the stage? It is still in RC. We bumped containerd maybe we consider bumping CNI as well!

Reply:
Yes, I agree to that. This part of the code is temporary, just to make it work. I am planning on bumping the CNI version to 0.9.x for k3s 1.22 and it should be done in 1.22.1 as well. I first need to merge https://github.com/rancher/plugins/pull/5

Reply:
What the heck I thought I migrated that repo over to k3s-io! :man_shrugging: